src/Riok.Mapperly/Descriptors/MappingBodyBuilders/BuilderContext/MembersMappingBuilderContext.cs

```diff
@@ -117,9 +115,25 @@ private HashSet<string> GetSourceMemberNames()
 
     private Dictionary<string, List<PropertyMappingConfiguration>> GetMemberConfigurations()
     {
-        return BuilderContext.Configuration.Properties.ExplicitMappings
-            .GroupBy(x => x.Target.Path.First())
-            .ToDictionary(x => x.Key, x => x.ToList());
+        var simpleMappings = BuilderContext.Configuration.Properties.ExplicitMappings.GroupBy(x => x.Target.Path.First());
+
+        var wildcardMappings = BuilderContext.Configuration.Properties.NestedMappings
+            .SelectMany(x =>
+            {
+                return BuilderContext.SymbolAccessor
+                    .GetAllAccessibleMappableMembers(Mapping.SourceType)
+                    .Where(i => i.Name == x.Source.Path.Last())
+                    .SelectMany(i => BuilderContext.SymbolAccessor.GetAllAccessibleMappableMembers(i.Type))
```

Comment:
The members included via `MapNestedProperties` should also be included in `_unmappedSourceMemberNames`  and should also emit a diagnostic if not mapped. This should also be tested.